src/Driver/DriverInterface.php

```diff
@@ -159,8 +167,8 @@ public function back();
     /**
      * Sets HTTP Basic authentication parameters.
      *
-     * @param string|false $user     user name or false to disable authentication
-     * @param string       $password password
```

Comment:
Unfortunately, the IDE has this style rule not enabled by default.
If it's preferred to have this sort of style, I can roll it back.